sw/legacy/common/rv_plic.c

```diff
@@ -7,7 +7,9 @@
 #include "dev_access.h"
 #include "sonata_system.h"
 
-#define NUM_IRQS 182
+#define NUM_IRQS 32
+
+#define I2C_ACQ_THRESHOLD_INTERRUPT 16
```

Comment:
Shouldn't this be called something like `I2C0_INTERRUPT` instead of `I2C_ACQ_THRESHOLD_INTERRUPT`, because we only have a single interrupt line per block now?

Reply:
Yes, have renamed this now.